### DISCUSSION TITLE
GraphQL showing data of speicific content type

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11148] [Q&A]
How i show specific data of specific content type based on some filter like category selection on page? with the help of GraphQL.

**Comment by Skrypt** [comment]
You generally create a Lucene or SQL Query that you expose with a Schema on the GraphQL endpoint.

**Reply by saleemhaq** [reply]
> You generally create a Lucene or SQL Query that you expose with a Schema on the GraphQL endpoint.

Can you please share with me the example how can i achieve.

**Reply by Skrypt** [reply]
https://docs.orchardcore.net/en/latest/docs/reference/modules/Queries/#graphql

**Reply by saleemhaq** [reply]
> https://docs.orchardcore.net/en/latest/docs/reference/modules/Queries/#graphql

@Skrypt can you please help me out in that: https://github.com/OrchardCMS/OrchardCore/discussions/11145
